pom.xml

```diff
@@ -139,6 +139,7 @@
          comes in via hbase-thirdparty hbase-shaded-netty-->
     <netty.hadoop.version>3.6.2.Final</netty.hadoop.version>
     <os.maven.version>1.6.1</os.maven.version>
+    <glassfish.el.version>3.0.1-b08</glassfish.el.version>
```

Comment:
how'd you come up with `3.0.1-b08` here? looking at the mvn install run from before the patch it looks like we end up getting `3.0.1-b11`.

Reply:
I took it from hbase repo, I believe I first tested with b11, it failed for a reason I don't recall. I can try again.

Reply:
I'm happy to match the hbase repo. :)